data/MI/incentives.json

```diff
@@ -0,0 +1,1400 @@
+    "end_date": 2023
+  },
+  {
+    "id": "MI-5",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250 rebate for 18-20.99 SEER/17-20 SEER2 ductless mini-split heat pump."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-6",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 350
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$350 rebate for 21.0 SEER/20 SEER2 or higher ductless mini-split heat pump."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-7",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 10
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$10 rebate for Programmable (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-8",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$50 rebate for Wi-Fi Enabled (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-9",
+    "authority_type": "utility",
+    "authority": "mi-consumers-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_consumersEnergy_residentialHeating,CoolingAndWaterHeatingProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 100
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$100 rebate for Wi-Fi Enabled (Must Replace an Existing Nonprogrammable Thermostat)."
+    },
+    "start_date": 2023,
+    "end_date": 2023
+  },
+  {
+    "id": "MI-10",
```

Comment:
Definitely not for this PR, but this kind of situation is starting to make me think we need a concept of a group of related incentives. Like these three weatherization incentives are actually just three variants of the same incentive. There's basically no way you'd claim more than one of them at once, but it's not a mutual exclusion relationship: we want to show all of them to the user, but ideally presented as a group of three variants rather than three separate incentives.

Just me thinking out loud, no action needed.

Reply:
Agreed! I think we're also not super consistent with how we represent these (sometimes they're separate incentives, sometimes they're listed as the same one)